pkg/kubelet/kubeletconfig/watch.go

```diff
@@ -94,7 +94,7 @@ func (cc *Controller) onUpdateNodeEvent(oldObj interface{}, newObj interface{})
 		utillog.Errorf("failed to cast old object to Node, couldn't handle event")
 		return
 	}
-	if !utilequal.ConfigSourceEq(oldNode.Spec.ConfigSource, newNode.Spec.ConfigSource) {
+	if !apiequality.Semantic.DeepEqual(oldNode.Spec.ConfigSource, newNode.Spec.ConfigSource) {
```

Comment:
Is this change related to the PR, or just something you are fixing along the way?

Reply:
Just fixing along the way. It's silly for me to write custom equality checks when we already have a utility that respects our API conventions.